components/eam/src/physics/cam/create_p3_lookupTable_1.f90-v4.1

```diff
@@ -0,0 +1,1812 @@
+PROGRAM make_p3_lookuptable1
```

Comment:
This file should be moved to cam/tools I think.  cam/src is only for source compiled in to the eam model itself.

Reply:
@AaronDonahue @jgfouca and whoever else, does this ever get run automatically? Can this just be moved to a new directory or will that break any tests?

Reply:
@brhillman , it does not look like it:

% cd scream/components
% git grep create_p3_lookup
eam/src/physics/cam/create_p3_lookupTable_1.f90-v4.1:!  Note:  compile with double-precision (pgf90 -r8 create_p3_lookupTable_1.f90)
eamxx/src/physics/p3/impl/p3_main_impl_part2.hpp:      // "create_p3_lookupTable_1.F90-v4.1".  All line numbers below reference this

Eamxx only has a comment referring to it.